distributed/client.py

```diff
@@ -1618,10 +1638,12 @@ def _handle_retried_key(self, key=None):
         if state is not None:
             state.retry()
 
-    def _handle_task_erred(self, key=None, exception=None, traceback=None):
+    def _handle_task_erred(
+        self, key=None, exception=None, traceback=None, erred_on=None
```

Comment:
Note: these defaults are there for historical reasons and very frequently you never have a use case for them.